modules/sql/src/main/resources/db/migration/V059__Niri_KeywordNames.sql

```diff
@@ -0,0 +1,47 @@
+---------------------------
+-- NIRI FITS Keyword Names
+---------------------------
+
+INSERT INTO e_fits_keyword_names(id, name)
+VALUES
+	('COADDS', 'COADDS'),
+	('EXPTIME', 'EXPTIME'),
+	('FILTER3', 'FILTER3'),
+	('FOCUSNAM', 'FOCUSNAM'),
+	('FOCUSPOS', 'FOCUSPOS'),
+	('FPMASK', 'FPMASK'),
+	('BEAMSPLT', 'BEAMSPLT'),
+	('WINDCOVR', 'WINDCOVR'),
+	('FRMSPCYCL', 'FRMSPCYC'),
+	('HDRTIMING', 'HDRTIMIN'),
+	('LNRS', 'LNRS'),
+	('MODE', 'MODE'),
+	('NDAVGS', 'NDAVGS'),
+	('PVIEW', 'PVIEW'),
+	('TDETABS', 'TDETABS'),
+	('TIME', 'TIME'),
+	('TMOUNT', 'TMOUNT'),
+	('UCODENAM', 'UCODENAM'),
+	('UCODETYP', 'UCODETYP'),
+	('VDDCL1', 'VDDCL1'),
+	('VDDCL2', 'VDDCL2'),
+	('VDDUC', 'VDDUC'),
+	('VDET', 'VDET'),
+	('VGGCL1', 'VGGCL1'),
+	('VGGCL2', 'VGGCL2'),
+	('VSET', 'VSET'),
+	('A_TDETABS', 'A_TDETAB'),
+	('A_TMOUNT', 'A_TMOUNT'),
+	('A_VDDCL1', 'A_VDDCL1'),
+	('A_VDDCL2', 'A_VDDCL2'),
+	('A_VDDUC', 'A_VDDUC'),
+	('A_VDET', 'A_VDET'),
+	('A_VGGCL1', 'A_VGGCL1'),
+	('A_VGGCL2', 'A_VGGCL2'),
+	('A_VSET', 'A_VSET');
+
+INSERT INTO e_dhs_keyword_names(keyword, name)
```

Comment:
What's the difference between DHS keywords and FITS keywords?

Reply:
FITS keywords follow the FITS keywords standard (i.e. up to eight capital letters). In the DHS, the keyword identifier has no such restrictions, although most of them have valid FITS keyword names. There is an additional field in the DHS dictionary for the actual FITS value that DHS uses when writing the keyword to the file. But inside Seqexec, all the keywords are FITS keywords (it had to handle DHS and GDS systems, so we use the more restrictive convention). The DHS table is actually used to translate the FITS keyword names to the ones DHS understand. It contains only the exceptions.